lunatrace/bsl/frontend/src/pages/project/dashboard/ScanTypesExplanation.tsx

```diff
@@ -40,34 +40,34 @@ export const ScanTypesExplanation: React.FC<{ setActiveTab: SetActiveTab; projec
             critical vulnerabilities in a comment on the PR.
           </Card.Subtitle>
         </ConditionallyRender>
-        <Card.Title>
```

Comment:
This card is just a little weird now since it is still titled "ways to take a snapshot"...I think it should be like "When is a snapshot created" for a title.  We could also break it down that commits to the default (master) branch will also trigger a build.